network-store-integration-test/src/test/java/com/powsybl/network/store/integration/NetworkStoreIT.java

```diff
@@ -4556,6 +4556,25 @@ public void testImportWithoutFlush() {
         }
     }
 
+    @Test
+    public void testImportWithPropertiesWithoutFlush() {
```

Comment:
Why `WithoutFlush` ?
We test with and without flush

Reply:
I agree, I repeated the name of the method above it without thinking properly
Changing it